swagger_templates/python/api_client.mustache

```diff
@@ -46,6 +46,14 @@ except ImportError:
     # for python2
     from urllib import quote, quote_plus
 
+# special handling of `long` (python2 only)
+try:
```

Comment:
Why not use [six](https://pythonhosted.org/six/#six.integer_types) for python2 and python3 support?

Reply:
The next step is to fully update the templates and SDK to the latest swagger_codegen code (which does indeed use six here). This is a minimal fix that incorporates the interim changes from upstream). I didn't want to do a halfway job, and the full update requires significantly more work.